docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
 title: "Using SSH"
 weight: 12
 description: |
-  One of the ways Platform.sh keeps things secure is by using SSH behind the scenes. Users can interact with their environment through a command shell, or push changes to the environment's Git repository, and both of these features rely on SSH.
+  Platform.sh uses Secure Shell (SSH) to help keep things secure. You can interact with your deployed environments or use the Platform.sh CLI, and both of these features rely on SSH.
```

Comment:
```suggestion
  Platform.sh uses Secure Shell (SSH) to help keep your project and applications secure. You can interact with your deployed environments or use the Platform.sh CLI, and both of these features rely on SSH.
```

Are we using things too much in this context?

Reply:
Yes, very much so.